README.md

```diff
@@ -3,79 +3,105 @@ Cancer Systems Biology, Section of Bioinformatics, Department of Health and Tech
 
-Otherwise it is possible to run the casestudy though Rstudio by running ./scripts/00_init.R
+1. clone our github repository into a local directory on your local machine:
 
-## FOLDERS:
+```
+git clone https://github.com/ELELAB/Moonlight2_DMA_basal_like.git
+cd Moonlight2_DMA_basal_like
+```
 
-### /data/
-The data generated by moonlight:
-Basal_FEA.rda, Basal_URA.rda, Basal_GRN.rda, Basal_PRA.rda, Basal_DMA.rda
-This data is stored at https://osf.io/eq9wj/
+2. create a virtual environment using conda activate it:
 
-#### /rawdata/
-The data that is not generated in this project, but was used as input to generate all other data and results in this project. 
-* Mutation from TCGA-BRCA "mutations.csv"
-* Differential Gene expression from TCGA-BRCA "BRCA_BRCA.Basal_dataDEGs.rda"
-* Normal counts of gene expression "BRCA_BRCA.Basal_dataFilt_HUGO.rda"
-* Driver genes predicted by GUST "GUST_BRCA.csv"
-* List of transcription factors from TRRUST "trrust_rawdata_human.tsv"
-* List of human kinases from Kinhub "kinhub.tsv"
-* List of PAM50 genes "pam50_list.txt"
-This data is stored at https://osf.io/eq9wj/
+```
+conda create --prefix ./env_Basal -c conda-forge r-base=4.2 r-pacman r-curl r-ragg
+conda activate ./env_Basal
+```
 
-### /scripts/
-The script 00_init.R loads and checks the prirequirements to run this project.
-All scripts are then run in the correct order.
-The script are numbered and the results from each are numbered accordingly.
+3. run the analyis:
 
-### /results/
-The three additional tables of data from DMA.This folder containts plots and tables used in the paper.The numbers refere to the script they are generated from.
+```
+$ bash ./Run_Basal.sh
+```
 
-### PAPER CONTENTS 
+This script will:
 
+1. Install in the environment all the necessary R packages, including Moonlight2R
+from [our GitHub repository](https://www.github.com/ELELAB/Moonlight2R)
+2. Download from [our OSF repository](https://osf.io/eq9wj/) the `data` folder
+that contains data required for the analysis (see below). This takes approximately
+700MB of disk space.
+3. perform the analysis
 
-Figure 1: Not in repository - created in biorender  
-Figure 2: 021_heatmap_complete.pdf  
-Figure 3: 022_Mediators_moonlight_heatmap.pdf  
-Figure 4: 032_promoter_moonlight_heatmap.pdf  
-Figure 5: 05_GO_KEGG_top_adjpval_OCG_TSG.png  
-Figure 6: 06_upsetplot_drivers.pdf  
+## Content of the downloaded directories
 
-Table 1: The CScape scores are derived from Rogers et al. 2020, see  website or original cscape scripts  
-Table 2: The summarised numbers are calculated in scripts/030_summarise.R  
-Table 3: 032_promoter_mutation_genes.csv  
-Table 4: 033_driver_TF.csv  
-Table S3: 06_overlap_moonlight_drivers_vs_ncg.csv  
-Table S4: 07_overlapping_genes_moonlight_ncg_gust.csv  
+### `./data`
+
+This contains the data generated by Moonlight2:
+`Basal_FEA.rda`, `Basal_URA.rda`, `Basal_GRN.rda`, `Basal_PRA.rda`, `Basal_DMA.rda`
+
+### `./data/raw_data`
```

Comment:
We decided to stick with the "rawdata" name and not "raw_data"

Reply:
fixed